src/pages/projects/forms/NetworkSelector.tsx

```diff
@@ -22,38 +22,89 @@ const NetworkSelector: FC<Props & SelectProps> = ({
   const { data: networks = [] } = useNetworks(project);
 
   const managedNetworks = networks.filter((network) => network.managed);
+  const hasAcls = managedNetworks.some((t) => t.config["security.acls"]);
 
   const getNetworkOptions = () => {
     const options = managedNetworks.map((network) => {
       return {
-        label: network.name,
+        label: (
+          <div className="label">
+            <span
+              title={network.name}
+              className="network-option name u-truncate"
+            >
+              {network.name}
+            </span>
+            <span title={network.type} className="network-option u-truncate">
+              {network.type}
+            </span>
+            {hasAcls ? (
+              <span
+                title={network.type}
+                className="network-option acls u-truncate u-align--right"
+              >
+                {network.config["security.acls"]?.split(",").length}
```

Comment:
We should have a fallback here for when no ACLs are on the network. The fallback should be either "0" or a "-".